FirstSolution/Part10Task7/Task.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Part10Task7
+{   
+    
+    // Написать 5ый метод, который будет принимать неопределенное число имен,
+    // и строго одно значение в качестве возраста. Метод должен выводить в
+    // консоль сообщение вида: "We are kolya, vasya and petya. And we are 50 years old"
+    
+    public class Task
+    {
+       private static void Main(string[] args)
+       {
+           Text( 40, "Alex", "Howard", "John");
```

Comment:
You should pass undefined number of names here

Reply:
Done